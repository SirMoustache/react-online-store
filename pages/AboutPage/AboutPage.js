/**
 * Absolute imports
 */
import React, { Fragment } from 'react';

const AboutPage = () => (
  <Fragment>
    <h1>Hello World from About Page</h1>
  </Fragment>
);

export default AboutPage;
