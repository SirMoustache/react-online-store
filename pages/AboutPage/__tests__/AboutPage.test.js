/**
 * Absolute imports
 */
import React from 'react';
import { render, cleanup } from '@testing-library/react';

/**
 * Local Components
 */
import AboutPage from '..';

const renderComponent = ({ ...rest } = {}) => render(<AboutPage {...rest} />);

describe('<AboutPage />', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
