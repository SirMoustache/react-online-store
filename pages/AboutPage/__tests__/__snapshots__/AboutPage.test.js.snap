// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> should match snapshot 1`] = `
<div>
  <h1>
    Hello World from About Page
  </h1>
</div>
`;
